### PR TITLE
chore: dockerize

### PR DESCRIPTION
**Pull request by MalinAhlberg**
This PR closes issue #40.

Usage:
`docker compose build`
`docker compose up`

The web page will be available at port `8000`.

### CODE REVIEW

**Comment by nanjiangshu** [comment]
Looks great @MalinAhlberg! I don't see any problem with the functionality of the web page.

**Comment by MalinAhlberg** [comment]
> Looks great @MalinAhlberg! I don't see any problem with the functionality of the web page.

Great, thanks! The PR is now undrafted.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
```
Comment (nanjiangshu) [inline docker-compose.yml:4]: Not quite sure why mysql is needed.

**Review by nanjiangshu — APPROVED** [review]
Great work @MalinAhlberg. I approve this PR despite a minor comment regarding the service with mysql.

**Review by inghylt — APPROVED** [review]
Great work! Just wondering, the code that is in dockerrun.sh, couldn't that be part of the Dockerfile?

**Comment by MalinAhlberg** [comment]
> Just wondering, the code that is in dockerrun.sh, couldn't that be part of the Dockerfile?

I was hoping so too, but did not get that to work. I think it's because the dir `/topcons` is a volume, and if data is changed in volumes during build steps, those changes will be discarded ([see this](https://docs.docker.com/engine/reference/builder/#notes-about-specifying-volumes)). But if there are prettier ways of doing this, I'd be happy to learn :)

File: docker-compose.yml (hunk 1 above)
Comment (MalinAhlberg) [reply docker-compose.yml]: Ah, no it seems it's not needed. We could keep it to use in future versions, but I think it's nicer to just delete it. It's removed in f7ac8b2. If you disagree, let me know and I'll readd it.

**Comment by inghylt** [comment]
@MalinAhlberg Ah interesting! But let's keep it as is and if we have any epiphanies in the future we can make updates then :)

**Comment by nanjiangshu** [comment]
> > Just wondering, the code that is in dockerrun.sh, couldn't that be part of the Dockerfile?
> 
> I was hoping so too, but did not get that to work. I think it's because the dir `/topcons` is a volume, and if data is changed in volumes during build steps, those changes will be discarded ([see this](https://docs.docker.com/engine/reference/builder/#notes-about-specifying-volumes)). But if there are prettier ways of doing this, I'd be happy to learn :)

I think one might use `ENTRYPOINT` to include the command in Dockerfile 
e.g. add this in the end of Dockerfile

```
ENTRYPOINT ["/topcons/dockerrun.sh"]
```

File: docker-compose.yml (hunk 1 above)
Comment (nanjiangshu) [reply docker-compose.yml]: Mysql is so far not used in web-topcons. I'm OK with either keep it or not.

**Comment by MalinAhlberg** [comment]
> I think one might use ENTRYPOINT to include the command in Dockerfile

Good suggestion! I did this in 028f293cb41082305dbbfd7b088a930ecc4592cf. It looks much better to me.

**Pull request merged**